myapp/app/views/users/show.html.erb

```diff
@@ -28,4 +32,29 @@
 
   <%= link_to I18n.t('action.list'), users_path, class:"btn btn-primary" %>
   <%= link_to I18n.t('action.edit'), edit_user_path(@user), class:"btn btn-primary" %>
+
+  <h4 class="mt-5"><%= @user.name %> <%= I18n.t('activerecord.models.task') %> <%= I18n.t('action.list') %></h4>
+  <table class="table table-striped table-hover">
+    <thead>
+      <tr>
+        <th class="text-center"><%= I18n.t(:id, scope: [:activerecord, :attributes, :task]) %></th>
+        <th class="text-center"><%= I18n.t(:name, scope: [:activerecord, :attributes, :task]) %></th>
+        <th class="text-center"><%= I18n.t(:deadline_at, scope: [:activerecord, :attributes, :task]) %></th>
+        <th class="text-center"><%= I18n.t(:created_at, scope: [:activerecord, :attributes, :task]) %></th>
+        <th class="text-center"><%= I18n.t(:updated_at, scope: [:activerecord, :attributes, :task]) %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @tasks.each do |task| %>
+        <tr>
+          <th class="text-center"><%= task.id %></th>
+          <th class="w-25"><%= task.name %></th>
+          <th class="text-center"><span class="<%= badge_class(task.status) %>"><%= I18n.t(task.status, scope: [:activerecord, :enums, :task, :status]) %></span></th>
+          <th class="text-center"><%= task.deadline_at&.strftime("%Y/%m/%d %H:%M")  %></th>
+          <th class="text-center"><%= task.created_at.strftime("%Y/%m/%d %H:%M")  %></th>
+          <th class="text-center"><%= task.updated_at.strftime("%Y/%m/%d %H:%M")  %></th>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
```

Comment:
- I changed to display a list of task of the users.

<img width="1506" alt="image" src="https://user-images.githubusercontent.com/37566073/225511289-abf18dff-08b5-4bfb-b41d-751fadc69279.png">